.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.swift]
+indent_style = space
+indent_size = 4
+tab_width = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = false
```

Comment:
My Xcode defaults to true and I think, that makes sense. Why did you set it to false?

Reply:
I have a setting (I think it’s the default settings in Xcode) to remove trailing whitespaces, _except for whitespace only lines_. This setting does not exist in the `.editorconfig` file, hence this value. More info about that in [this issue](<https://github.com/editorconfig/editorconfig/issues/520>).

That being said you can do whatever you want. It’s your project, it’s your code-style! Feel free to modify the file I propose however you want before merging; for me the most important part is that the file _exists_. 😉

Reply:
Ah, I see. No, I think you are right on this. I want to keep whitespace-only lines.